src/data_types/credential.rs

```diff
@@ -73,14 +75,71 @@ impl Validatable for Credential {
 #[derive(Clone, Debug, PartialEq, Eq, Deserialize, Serialize)]
 pub struct CredentialInfo {
     pub referent: String,
-    pub attrs: ShortCredentialValues,
+    pub attrs: RawCredentialValues,
     pub schema_id: SchemaId,
     pub cred_def_id: CredentialDefinitionId,
     pub rev_reg_id: Option<RevocationRegistryDefinitionId>,
     pub cred_rev_id: Option<String>,
 }
 
-pub type ShortCredentialValues = HashMap<String, String>;
+#[derive(Clone, Debug, Default, PartialEq, Eq, Deserialize, Serialize)]
+pub struct RawCredentialValues(pub HashMap<String, String>);
+
+#[cfg(feature = "zeroize")]
+impl Drop for RawCredentialValues {
+    fn drop(&mut self) {
+        self.zeroize();
+    }
+}
+
+#[cfg(feature = "zeroize")]
+impl Zeroize for RawCredentialValues {
+    fn zeroize(&mut self) {
+        for attr in self.0.values_mut() {
+            attr.zeroize();
+        }
+    }
+}
+
+impl Validatable for RawCredentialValues {
```

Comment:
just curious what is the reason for change from Short to Raw is this because we need richer data for the flex creds?

Reply:
I just considered name `short` not clear and confusing. can be reverted